finagle-thrift/src/main/ruby/lib/finagle-thrift/trace.rb

```diff
@@ -100,13 +100,14 @@ class Flags
   end
 
   class SpanId
-    HEX_REGEX = /^[a-f0-9]{16}$/i
+    HEX_REGEX = /^[a-f0-9]{16,32}$/i
```

Comment:
@jcarres-mdsol I think the https://github.com/openzipkin/zipkin-ruby relies on this class, still. lemme know if my adjustment is not idiomatic